platform/src/main/resources/anchor-docker-compose-config.yaml

```diff
@@ -234,7 +234,7 @@ assets:
                       "iso4217:USD"
                   ]
               },
-              "sep24_enabled": false,
```

Comment:
This is the conflict with this PR: https://github.com/stellar/java-stellar-anchor-sdk/pull/677